src/states/adult.rs

```diff
@@ -435,6 +432,8 @@ impl Base for Adult {
         // Need to collect IDs first so that self is not borrowed via the iterator
         //
         // WIP: this is probably out of date? How else do we know which our section members are?
+        // WIP: once ConnectionRequest/ConnectionResponse is removed this whole function can
```

Comment:
Not sure about that, maybe.
However, when Elder are demoted to Adults, they should forward RoutingMessages to Elders.